package.json

```diff
@@ -110,7 +110,8 @@
         "ansi-regex": "^5.0.1",
         "shell-quote": "^1.7.2",
         "node-forge": "^1.0.0",
-        "node-fetch": "^3.1.1"
+        "node-fetch": "^3.1.1",
+        "follow-redirects": "^1.14.8"
```

Comment:
Trenger denne å ligge i dependency-lista egentlig? Jeg det var transitiv avhengighet, ikke noe vi bruker direkte

Reply:
Det er i `resolutions`, men enig i at det er for mange dependencies der.

Reply:
EDIT: Bare ignorer, så feil :)